### PR TITLE
ES-831427:  Convert_Word_to_Image_in_Azure_App_Service

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Main-page and Sub-page added for Convert Word to Image in Azure App Service

Regards
Venkatesh

### CODE REVIEW

File: File-Formats/DocIO/Convert-Word-Document-to-Image-Azure.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+# Convert Word to Image in Azure Platform 
+
+Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit, and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, **convert a Word document to a Image in Azure services** within a few lines of code.
+
+N> If this is your first time working with Azure, please refer to the dedicated Azure development resources. This section explains how to convert Word documents to Image in C# using the .NET Word (DocIO) library in Azure. 
+
+## Prerequisites 
+* An active **Microsoft Azure subscription** is required. If you don’t have one, please create a free account before starting.
+
+## NuGet Packages Required
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-Azure.md:18]: Change it as Azure Services.

Also mention v1 and v4 functions.

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+title: Convert Word document to Image in Azure App Service on Windows | Syncfusion
+description: Convert Word document to Image in Azure App Service on Windows using .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to Image in Azure App Service on Windows
+
+Syncfusion  DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in Azure App Service on Windows**
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md:11]: Remove unwanted spaces, empty lines.
Add full stop properly.

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+title: Convert Word document to Image in Azure App Service on Linux | Syncfusion
+description: Convert Word document to Image in Azure App Service on Linux using .NET Core Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Convert Word to Image in Azure App Service on Linux
+
+Syncfusion  DocIO is a [.NET Core Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, edit and **convert Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **convert a Word document to Image in Azure App Service on Linux**
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md:11]: Remove unwanted empty space.
Add full stop.

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+![Select the publish target as Azure](Azure_Images/App_Service_Linux/Publish_Target_WordtoPDF.png)
+
+Step 4: Select the Specific target as **Azure App Service (Linux)**.
+![Select the publish target](Azure_Images/App_Service_Linux/Specific_Target_WordtoPDF.png)
+
+Step 5: To create a new app service, click **Create new** option.
+
+![Click create new option](Azure_Images/App_Service_Linux/Create_New_App_Service_WordtoPDF.png)
+
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md:177]: Remove unwanted empty lines

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+![Install Syncfusion.DocIORenderer.Net.Core Nuget Package](Azure_Images/App_Service_Linux/Syncfusion_Nuget_Package_WordtoPDF.png)
+
+Step 5: Add a new button in the **Index.cshtml** as shown below.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+@{Html.BeginForm("WordToImage", "Home", FormMethod.Post, new { enctype = "multipart/form-data" });
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md:34]: Remove unwanted empty lines in code snippet.

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md [hunk 6]
```diff
@@ -0,0 +1,200 @@
...
+![Select the publish target as Azure](Azure_Images/App_Service_Linux/Publish_Target_WordtoPDF.png)
+
+Step 4: Select the Specific target as **Azure App Service (Windows)**.
+![Select the publish target](Azure_Images/App_Service_Windows/Specific_Target_WordtoPDF.png)
+
+Step 5: To create a new app service, click **Create new** option.
+
+![Click create new option](Azure_Images/App_Service_Linux/Create_New_App_Service_WordtoPDF.png)
+
+
```
Comment (Suriya-Balamurugan) [inline File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md:177]: Remove unwanted empty lines

**Review by Suriya-Balamurugan — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Convert-Word-Document-to-Image-Azure.md (hunk 1 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-Azure.md:18]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md (hunk 6 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md (hunk 5 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md (hunk 4 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md (hunk 3 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Linux.md]: updated

File: File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md (hunk 2 above)
Comment (Venkateshmuruganandam) [reply File-Formats/DocIO/Convert-Word-Document-to-Image-in-Azure-App-Service-Windows.md]: updated

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Review by MohanaselvamJothi — APPROVED** [review]

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**